src/first/FastFileSearch.java

```diff
@@ -0,0 +1,23 @@
+package first;
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class FastFileSearch implements Runnable {
+    Thread thrd;
+    SuitableFile file;
+    static String phrase;
+
+    @Override
+    public void run() {
+        file.setSuitable(phrase);
```

Comment:
maybe should be better if this function will be return boolean value? And you could store this flag in FastFileSearch.